iamy/models.go

```diff
@@ -113,6 +113,15 @@ type Role struct {
 	Policies                 []string        `json:"Policies,omitempty"`
 }
 
+type InstanceProfile struct {
```

Comment:
we probably want to fix the alignment on this one

```go
type InstanceProfile struct {
	iamService `json:"-"`
	Roles      []string `json:"Roles,omitempty"`
}
```

Reply:
It's aligned if your tabs are set to 8 :-)